docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -0,0 +1,268 @@
+
+1. Get a timestamp. YCQL doesn't have a now() function, so use a command such as one of the following:
+
+    ```sh
+    # Ruby: remove the decimal point before using the timestamp
+    $ ruby -e 'puts Time.now.to_f'
+    ```
+
+    ```sh
+    # Python: remove the decimal point before using the timestamp
+    $ python -c 'import datetime; print datetime.datetime.now().strftime("%s.%f")'
+    ```
+
+    ```sh
+    # Linux and some other systems (but not macOS): use the timestamp as-is
+    $ date +%s%N | cut -b1-15
+    ```
+
+1. Add a row for employee 9999 to the table:
+
+    ```sql
+    insert into employees (employee_no, name, department, salary) values (9999, 'Wrong Name', 'Marketing', 10000);
+
+    select * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            9999 |     Wrong Name |  Marketing |  10000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (5 rows)
+    ```
+
+1. List snapshots, at a terminal prompt:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                         State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    No snapshot restorations
+    ```
+
+1. Restore the latest snapshot to the timestamp you obtained before you deleted the data, at a terminal prompt:
+
+    ```sh
+    $ bin/yb-admin restore_snapshot bb5fc435-a2b9-4f3a-a510-0bacc6aebccf 1617670679185100
+
+1. Next, verify the restoration is in `RESTORED` state:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                     State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    Restoration UUID                      State
+    bd7e4e52-b763-4b95-87ce-9399e1ac206e  RESTORED
+    ```
+
+1. In the YCQL shell, verify the data is restored, and there is no row for employee 9999:
+
+    ```sql
+    ycqlsh:pitr> select * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+### Restore from a relative time
+
+In addition to restoring to a particular timestamp, you can also restore to a relative time, such as "ten minutes ago". In this example, you'll delete some data from the existing `employees` table, then restore the state of the database to what it was five minutes prior.
+
+When you specify a relative time, you can specify any or all of _days_, _hours_, and _minutes_. For example:
```

Comment:
For entering in these times, we support:
1. ISO 8601: '3d 4h 5m 6s'
2. Abbreviated Postgres: '3 d 4 hrs 5 mins 6 secs'
3. Traditional Postgres: '3 days 4 hours 5 minutes 6 seconds'
4. SQL Standard: 'D H:M:S'

I'm not sure if you want to comment that here.

Reply:
Excellent! Yes, added.